test/fixtures/commits_after_ci.json

```diff
@@ -0,0 +1,16 @@
+  {
+    "commits": [{
+      "commit": {
+        "committedDate": "2017-10-25T11:27:02Z",
```

Comment:
Can you extend this fixtures a bit and add one more commits? Just so we can see how it's gonna look like when there are multiple new commits.

Reply:
ok